workers/notifier/src/test/kotlin/notifier/NotifierOrtResultGeneratorTest.kt

```diff
@@ -202,6 +203,14 @@ private class ResultGeneratorTestHelper {
         every { getEvaluatorJobForOrtRun(ORT_RUN_ID) } returns null
         every { getScannerJobForOrtRun(ORT_RUN_ID) } returns null
         every { getReporterJobForOrtRun(ORT_RUN_ID) } returns null
+        every { generateOrtResult(ortRun, failIfRepoInfoMissing = false) } returns OrtResult.EMPTY.copy(
```

Comment:
Can the other `every` declarations now be dropped? I think, on the mock only the new `generateOrtResult()` function will be called.

Reply:
I am not sure if I understand correct. The `OrtRunService` is completely mocked, and it is used by test cases to check if 3 types of notifier-specific _labels_ are added. As this `generateOrtResult` function is called every time in order to add the common _labels_, I **have** to mock it, else the test does not find the function on the OrtRunService.

Reply:
The `generateOrtResult()` has to be mocked for sure. I meant the other stubbing declarations, e.g. for `getXXXJobForOrtRun()`. They were necessary before, as they used to be called by the result generator. But now the result generator only calls `generateOrtResult()` and no longer the other functions.